keps/sig-network/4004-deprecate-kube-proxy-version/README.md

```diff
@@ -204,11 +204,23 @@ N/A
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-N/A
+  I've tried to simulate this manually by running (using `local-cluster-up.sh`with `PRESERVE_ETCD=true`):
```

Comment:
> Are there any tests for feature enablement/disablement?

[...] we will manually confirm that restarting kubelet with the [..]

has that happened? Please update

Reply:
added.

Reply:
This is great. However, I was refering to the question from line 190 saying: "we will manually confirm that restarting ...".

Please update that saying that it was already manually tested with the steps described below.

Reply:
Fixed, please review whether it meets your expectations.